rosetta-source/src/main/rosetta/event-common-type.rosetta

```diff
@@ -250,6 +252,16 @@ type SettlementOrigin: <"Defines the origin to the transfer as a reference for l
 type TradeIdentifier extends Identifier: <"Defines a trade identifier as a special case of the generic identifier type, that also includes the trade identifier class.">
 	identifierType TradeIdentifierTypeEnum (0..1) <"The enumerated classification of the identifier. Optional as a trade identifier may be party-specific, in which case it may not correspond to any established classification.">	
 
+type Valuation: <"Defines the value of an investment, asset, or security">
+    amount Money (1..1) <"Current value of the outstanding contract">
+    timestamp zonedDateTime (1..1) <"Date and time of the last valuation marked to market, provided by the central counterparty (CCP) or calculated using the current or last available market price of the inputs.">
+    method ValuationTypeEnum (1..1) <"Method used for the valuation of the transaction by the valuation party.">
+    source ValuationSourceEnum (1..1) <"Source of the valuation of the transaction by the valuation party.">
```

Comment:
Both `method` and `source` should be (0..1) because of the `required choice` rule